contracts/BSCValidatorSet.sol

```diff
@@ -131,6 +145,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   event directTransferFail(address payable indexed validator, uint256 amount);
   event deprecatedDeposit(address indexed validator, uint256 amount);
   event validatorDeposit(address indexed validator, uint256 amount);
+  event validatorSharing(address indexed validator, uint256 amount);
```

Comment:
`rewardShared` is a bit better than `validatorSharing`? Seems we follow a pattern to define event like `event somethingDone ...`,  some word like `Sharing` which means `doing` is not a proper word here.